### PR TITLE
hexdump: add page

### PR DESCRIPTION
**Pull request by ZeroDayOwl**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known): 2.37.2**

### CODE REVIEW

**Comment by ZeroDayOwl** [comment]
@navarroaxel Hi! Could you kindly review my PR?

**Comment by navarroaxel** [comment]
Hi @ZeroDayOwl this page already exists in the `common` directory because the command is available in macOS too https://github.com/tldr-pages/tldr/blob/main/pages/common/hexdump.md

**Comment by kbdharun** [comment]
> Hi @ZeroDayOwl this page already exists in the `common` directory because the command is available in macOS too https://github.com/tldr-pages/tldr/blob/main/pages/common/hexdump.md

IMO, This page looks good but it doesn't have the last 2 commands in the [main page](https://github.com/tldr-pages/tldr/blob/main/pages/common/hexdump.md). I think the PRs author can update the main page in common with this one along with the 2 missing commands.

File: pages/linux/hexdump.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# hexdump
+
+> Display file contents in hexadecimal, decimal, octal, or ascii.
+> More information: <https://man7.org/linux/man-pages/man1/hexdump.1.html>.
+
+- Display the input offset in hexadecimal, followed by three columns of data in octal, per line:
+
+`hexdump -b {{path/to/file}}`
```
Comment (EmilySeville7cfg) [inline pages/linux/hexdump.md:8]:
```suggestion
- Display the input offset in hexadecimal, followed by three columns of data in octal, per line:

`hexdump --one-byte-octal {{path/to/file}}`
```

Mandatory fix: [long option must be used if it's portable](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#option-syntax).

File: pages/linux/hexdump.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+- Display the input offset in hexadecimal, followed by same sixteen bytes in `%_p` format enclosed in `|` characters:
+
+`hexdump --canonical {{path/to/file}}`
+
+- Interpret only specified length bytes of the input:
+
+`hexdump --length {{7}} {{path/to/file}}`
```
Comment (EmilySeville7cfg) [inline pages/linux/hexdump.md:16]:
```suggestion
- Interpret only specified length bytes of the input:

`hexdump --length {{1..infinity}} {{path/to/file}}`
```

Optional fix: [range syntax can be used](https://github.com/tldr-pages/tldr/pull/9646).
Note: I am not sure whether length can't be 0.

File: pages/linux/hexdump.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+- Skip offset bytes from the beginning of the input and interpret:
+
+`hexdump --skip {{5}} {{path/to/file}}`
```
Comment (EmilySeville7cfg) [inline pages/linux/hexdump.md:20]:
```suggestion
- Skip offset bytes from the beginning of the input and interpret:

`hexdump --skip {{1..infinity}} {{path/to/file}}`
```

The same issue as above.

File: pages/linux/hexdump.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+
+- Display all data in hexadecimal without skipping any output lines:
+
+`hexdump -v {{path/to/file}}`
```
Comment (EmilySeville7cfg) [inline pages/linux/hexdump.md:24]:
```suggestion
- Display all data in hexadecimal without skipping any output lines:

`hexdump --no-squeezing {{path/to/file}}`
```

Mandatory fix: [long option must be used if it's portable](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#option-syntax).

**Review by EmilySeville7cfg — CHANGES_REQUESTED** [review]
Mandatory fixes found, changes requested.
Thanks for a contribution! 🚀

**Pull request closed without merging**